config/capsule.yaml

```diff
@@ -0,0 +1,155 @@
+# foreman-installer main configuration file
+# note current configuration is written to foreman-installer.yaml every time foreman-install is run
+
+## Installer configuration
+# Path to answer file
+:answer_file: /etc/foreman/installer-scenarios.d/capsule-answers.yaml
+:installer_dir: /usr/share/foreman-installer
+# Uncomment if you want to load puppet modules from a specific path, $pwd/modules is used by default
+:module_dirs: /usr/share/foreman-installer/modules
+
+## Useful for development, e.g. when you want to move log files to local directory
+:log_dir: '/var/log/foreman-installer'
+:log_name: 'foreman-installer.log'
+:log_level: :debug
+
+# Change if you want to debug default answers for you modules, this directory holds default answers
+# :default_values_dir: /tmp
+
+## Advanced configuration - if not set it's ignored
+# :log_owner: root
+# :log_group: root
+# :config_header_file:
+# :dont_save_answers:
+
+# If using the Debian ruby-kafo package, uncomment this
+# :kafo_modules_dir: /usr/share/foreman-installer/modules
+
+## Kafo tuning, customization of core functionality
+:name: Capsule
+:description: Install a stand-alone Capsule.
+# :no_prefix: false
+:order:
+  - certs
+  - foreman
+  - capsule
+  - foreman_proxy
+  - puppet
+:low_priority_modules:
+  - foreman_proxy_plugin
+  - foreman_compute
+  - foreman_plugin
+
+# The mapping hash provides Kafo with the information to find plugin classes
```

Comment:
@ekohl, Agreed.  It would be nice to minimize some of the duplication in the configuration files; however, that is not yet supported by kafo.  @mbacovsky is aware of the current duplication and has reducing it as one of goals for kafo in the future, it just isn't there yet.  Ideally, we'd really like to have the ability to add katello to existing foreman sooner than later, so we are working within current constraints.